deploy/charts/openebs-monitoring/templates/podmonitors.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.podMonitors }}
+{{- range $openebsEngine, $fields := .Values.podMonitors }}
+{{- if (and (hasKey $fields "enabled") ($fields.enabled)) }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ template "openebs-monitoring.fullname" $ }}-{{ $openebsEngine }}
+  namespace: {{ template "openebs-monitoring.namespace" $ }}
+  labels:
+    app: {{ template "openebs-monitoring.name" $ }}-{{ $openebsEngine }}
+{{ include "openebs-monitoring.labels" $ | indent 4 }}
+spec:
+{{- if (and (hasKey $fields "labelSelector") ($fields.labelSelector)) }}
```

Comment:
We can avoid multiple if-else statement by assigning default values in values.yaml.

Reply:
Done